Documentation/Books/AQL/Examples/RemoveVertex.md

```diff
@@ -0,0 +1,54 @@
+Remove Vertex
+=============
+
+Deleting vertices with associated edges is currently not handled via AQL while the [graph management interface](../../Manual/Graphs/GeneralGraphs/Management.html#remove-a-vertex) and the [REST API for the graph module](../../HTTP/Gharial/Vertices.html#remove-a-vertex) offer a vertex deletion functionality.
```

Comment:
Try to wrap lines at 80 characters. This is also possible within links to some extent. I successfully tested the following pattern:

```
Some [link label here, possibly quite long
](https://some.long.link/wrapped_to_a_new_line/note_that_the_closing_square_bracket_is_on_this_line)
```

Reply:
I vote against this. 
It makes writing testing functions unneccesarily hard.

Reply:
We already use this in some places. We will need to change it then.

Reply:
yes, we should.